js/gated-content/src/store/modules/auth/personify.js

```diff
@@ -0,0 +1,65 @@
+import client from '@/client';
+
+export default {
+  state: {
+    config: null,
+  },
+  actions: {
+    async personifyAuthorize(context) {
+      return client.post(context.getters.getPersonifyConfig.api_login_check)
+        .then((r) => {
+          // Check user data.
+          if (r.data.user && r.data.user.email) {
+            context.dispatch('authorize', r.data.user);
+          } else {
+            // Get Personify login URL and redirect to login page.
+            client({
```

Comment:
We can construct link on backend side.
There is a method at gc plugin, called getDataForApp().